backend/src/main/java/com/bakdata/conquery/models/forms/managed/RelativeFormQueryPlan.java

```diff
@@ -209,61 +146,20 @@ private void assertResultWidth(EntityResult subResult, int resultWidth) {
 	}
 
 
-	private boolean hasCompleteDateContexts(List<DateContext> contexts) {
-		if (contexts.isEmpty()) {
-			return false;
-		}
-
-		if (featurePlan.getAggregatorSize() <= 0 || outcomePlan.getAggregatorSize() <= 0) {
-			// Otherwise, if only features or outcomes are given check the first date context. The empty feature/outcome query
-			// will still return an empty result which will be merged with to a complete result.
-			return Resolution.COMPLETE.equals(contexts.get(0).getSubdivisionMode());
-		}
-
-		// We have features and outcomes check if both have complete date ranges (they should be at the beginning of the list)
-		return contexts.size() >= 2
-			   && Resolution.COMPLETE.equals(contexts.get(0).getSubdivisionMode())
-			   && Resolution.COMPLETE.equals(contexts.get(1).getSubdivisionMode())
-			   && !contexts.get(0).getFeatureGroup().equals(contexts.get(1).getFeatureGroup());
-	}
-
 	private static FormQueryPlan createSubQuery(ArrayConceptQueryPlan subPlan, List<DateContext> contexts, FeatureGroup featureGroup) {
 		List<DateContext> list = new ArrayList<>(contexts);
 		list.removeIf(dctx -> dctx.getFeatureGroup() != featureGroup);
 
-		return new FormQueryPlan(list, subPlan);
-	}
-
-	private void setFeatureValues(Object[] result, Object[] value) {
-		// copy everything up to including index
-		System.arraycopy(value, 0, result, 0, EVENTDATE_POS + 1);
-		// copy daterange
-		result[getFeatureDateRangePosition()] = value[SUB_RESULT_DATE_RANGE_POS];
-		System.arraycopy(value, SUB_RESULT_DATE_RANGE_POS + 1, result, getFirstAggregatorPosition(), featurePlan.getAggregatorSize());
-	}
-
-	private void setOutcomeValues(Object[] result, Object[] value) {
-		// copy everything up to including index
-		for (int i = 0; i <= EVENTDATE_POS; i++) {
-			if (result[i] != null) {
-				// Skip cells that where already filled from the feature result (in complete merge)
-				continue;
-			}
-			result[i] = value[i];
-		}
-		// copy daterange
-		result[getOutcomeDateRangePosition()] = value[SUB_RESULT_DATE_RANGE_POS];
-		System.arraycopy(value, SUB_RESULT_DATE_RANGE_POS + 1, result, getFirstAggregatorPosition()
-																	   + featurePlan.getAggregatorSize(), outcomePlan.getAggregatorSize());
+		return new FormQueryPlan(list, subPlan, true);
 	}
 
 	public List<Aggregator<?>> getAggregators() {
-		return ImmutableList.copyOf(Iterables.concat(featurePlan.getAggregators(), outcomePlan.getAggregators()));
+		return ImmutableList.copyOf(featurePlan.getAggregators());
 	}
 
 	@Override
 	public boolean isOfInterest(Entity entity) {
-		return query.isOfInterest(entity) || featurePlan.isOfInterest(entity) || outcomePlan.isOfInterest(entity);
+		return query.isOfInterest(entity) || featurePlan.isOfInterest(entity);
```

Comment:
Wenn die Entity nicht of Interest ist, wird doch auch nichts emitiert oder? Dann könnte das doch sogar eine Verundung sein?

Reply:
Verundung ist denke ich nicht richtig, weil eine Entity im Ergebnis eine Zeile bekommen soll, wenn sie in `query` drinne ist, auch wenn sie für `featurePlan` uninteressant ist.

Ich überlege ob einfacht `return query.isOfInterest(entity);` ausreichen würde

Reply:
Hast du dir das angeschaut?

Reply:
Ja ist besser und sauberer wenn die entity schon gleich rausfliegt wenn sie nicht für die `query` interessant ist. Ansonsten fliegt sie später raus, so wie es bisher geschehen ist.